### ISSUE TITLE
No FileSets for 3.20~40

### ISSUE THREAD

**Issue by drtinaz** [kwindrem/ExtTransferSwitch#13]
ExtTransferSwitch v2.38 getting no FileSets on beta 3.20~40

**Comment by kwindrem** [comment]
I can not reproduce this issue. I would need logs to troubleshoot further.

**Comment by drtinaz** [comment]
I looked at the v3.20~40 fileset folder. All of the ORIG files and the replacement files are present, except that there is also a start stop.py.no_replacement file (a startstop.py file is also present and is not empty) and a INCOMPLETE file. This is AFTER I have disabled automatic install, deleted the fileset, and initiated a new download of the Ext package. Attached is an image of the fileset contents after download with auto install disabled. I verified that the v3.20~40 fileset was deleted before re downloading the Ext package. The fileset shouldn't contain the INCOMPLETE without attempting an install should it? Am I missing something?

![Screenshot_20240125-214833.png](https://github.com/kwindrem/ExtTransferSwitch/assets/120696745/d5a5567f-17e4-4b9e-97b0-ef1703f90f1a)
